opennebula/defaults.yaml

```diff
@@ -99,10 +74,69 @@ kernel:
       controller_sshpubkeys: {}
 os_family:
   Debian:
+    oneadmin:
+      optional_groups:
+        - kvm
+        - libvirt
+        - ssl-cert
+    controller:
+      pkgs:
+        - opennebula
+        - opennebula-tools
+        - mysql-client
```

Comment:
Please comment this line out. I think it's better to install no DB libs/ driver by default.